bears/java/CheckstyleBear.py

```diff
@@ -10,6 +10,13 @@
     "geosoft": "http://geosoft.no/development/geosoft_checks.xml"}
 
 
+def invalid_configuration(checkstyle_configs, use_spaces, indent_size):
+    if checkstyle_configs != 'google':
+        return
```

Comment:
rather `return False` right? Also you could just `return checkstyle_configs == 'google' and ...` to make it a oneliner, right?